### PR TITLE
Blog post about plans for windows installer

### PR DESCRIPTION
**Pull request by rgaiacs**
@naupaka @twitwi @k8hertweck @maneesha  could you review this?

@tracykteal @ErinBecker let me know if this should be cross-post by Data Carpentry.

cc @jduckles

### CODE REVIEW

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+was about "nano not found",
+a bug on our installer that we never manage to trace.
+The [second thread](http://lists.software-carpentry.org/pipermail/discuss/2017-March/005140.html)
+was about some "misbehaviour" of nano on Windows
+and the suggestion to use [Atom](https://atom.io/) as the default text editor.
+The suggestion to use Atom
+started a long discussion where instructors manifest their reasons to use nano
+and why Atom is "evil" which lead us to start investigating ways to install nano properly on Windows.
+On May 3rd,
+Kate Hertweck, Maneesha Sane, Naupaka Zimmerman, Rémi Emonet and I met online
```
Comment (rgaiacs) [inline _posts/2017/05/2017-05-04-windows-installer.md]: Let me know if I should include links to your profile.

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Plan
+
+We will work to use [Git for Windows SDK](https://github.com/git-for-windows/git/wiki/Technical-overview)
+to compile our own installer that will include
+bash, Git, nano, SQLite and make.
+In the future we can work to include man pages and Jekyll.
+
+## Timeline
+
+- YYYY-MM-DD: Publish Plan
```
Comment (rgaiacs) [inline _posts/2017/05/2017-05-04-windows-installer.md]: Need to fill the dates.

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+- YYYY-MM-DD: Beta version of the installer is release
+- YYYY-MM-DD: Instructors test the installer
+- YYYY-MM-DD: First version of the installer is release
+- YYYY-MM-DD: Workshop template is update to use the installer
+- YYYY-MM-DD: Report back from first 3 workshops using the new installer
```
Comment (rgaiacs) [inline _posts/2017/05/2017-05-04-windows-installer.md]: @maneesha @kariljordan would be possible for you do this one?

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+## Acknowledgement
+
+Thanks very much for everyone that contribute with the discussion
```
Comment (rgaiacs) [inline _posts/2017/05/2017-05-04-windows-installer.md]: @k8hertweck Please check the language.

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+on the mailing list, GitHub issues and other places.
+We had a great resource for anyone that want to investigate
+other options for different projects.
+
+## FAQ
```
Comment (rgaiacs) [inline _posts/2017/05/2017-05-04-windows-installer.md]: Maybe I missing one question or two.

**Comment by tracykteal** [comment]
Thanks @rgaiacs since it's about the SWC installer, it doesn't need to be cross posted.

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+related with nano on Windows machine.
+The [first thread](http://lists.software-carpentry.org/pipermail/discuss/2017-March/005085.html)
+was about "nano not found",
+a bug on our installer that we never manage to trace.
+The [second thread](http://lists.software-carpentry.org/pipermail/discuss/2017-March/005140.html)
+was about some "misbehaviour" of nano on Windows
+and the suggestion to use [Atom](https://atom.io/) as the default text editor.
+The suggestion to use Atom
+started a long discussion where instructors manifest their reasons to use nano
+and why Atom is "evil" which lead us to start investigating ways to install nano properly on Windows.
```
Comment (naupaka) [inline _posts/2017/05/2017-05-04-windows-installer.md]: not sure I'd say 'evil' here.

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Plans for Windows Installer"
+date: 2017-05-04
+time: "16:00:00"
+category: ["Community","Tools","Teaching"]
+---
+
```
Comment (naupaka) [inline _posts/2017/05/2017-05-04-windows-installer.md:9]: Might be worth adding in something here about what the SWC needs are: a (preferably) command-line text editor for use (primarily) during the shell and git modules of SWC workshops that works across all platforms (Windows, macOS, *nix) and that is easy or transparent to install along with the other tools we ask learners to have before showing up.

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+The suggestion to use Atom
+started a long discussion where instructors manifest their reasons to use nano
+and why Atom is "evil" which lead us to start investigating ways to install nano properly on Windows.
+On May 3rd,
+Kate Hertweck, Maneesha Sane, Naupaka Zimmerman, Rémi Emonet and I met online
+to draft a plan---taking in consideration all the feedback that was provided in the past month---to solve the problem with nano on Windows.
+
+## Plan
+
+We will work to use [Git for Windows SDK](https://github.com/git-for-windows/git/wiki/Technical-overview)
```
Comment (naupaka) [inline _posts/2017/05/2017-05-04-windows-installer.md]: I'd soften this a bit - something like: "Based on the community input, exploration of a range of options by the committee, and the balance between ease of installation, ease of maintenance, and coherence with the goals of SWC, we have agreed to move forward by working to use Git for Windows SDK..."

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+2. Which version of Windows the installer will support?
+
+   The installer will support Windows 7, Windows 8 and Windows 10.
+   The [end of extended support to Windows 7 by Microsoft is January 14, 2020](https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet)
+   and because we don't have data about how many learners are using Windows 7
+   we believe would be unfair to not include Windows 7
+   at the first release of our installer.
+
+3. Would I be allow to use my own installer for my workshop?
```
Comment (twitwi) [inline _posts/2017/05/2017-05-04-windows-installer.md]: -> allowed

**Comment by jduckles** [comment]
@rgaiacs please let me know when this is ready to go.

**Comment by rgaiacs** [comment]
@k8hertweck Could you call me before I work on the suggested changes?

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+to draft a plan---taking in consideration all the feedback that was provided in the past month---to solve the problem with nano on Windows.
+
+## Plan
+
+We will work to use [Git for Windows SDK](https://github.com/git-for-windows/git/wiki/Technical-overview)
+to compile our own installer that will include
+bash, Git, nano, SQLite and make.
+In the future we can work to include man pages and Jekyll.
+
+## Timeline
```
Comment (rgaiacs) [inline _posts/2017/05/2017-05-04-windows-installer.md]: @k8hertweck I would like to discuss this with you.

File: _posts/2017/05/2017-05-04-windows-installer.md (hunk 2 above)
Comment (k8hertweck) [reply _posts/2017/05/2017-05-04-windows-installer.md]: I think it's important to get this post out soon, but until we have someone committed to working on it, I'm hesitant to put dates. Can we say something like "We're in the process of identifying someone to develop a testable version of this installer. After it is complete, we will make it available for the community to test for a week, update the workshop template to include instructors for the installer, and field test it in workshops. We hope to have  reports back from those workshops within 4-6 weeks."

File: _posts/2017/05/2017-05-04-windows-installer.md (hunk 3 above)
Comment (kariljordan) [reply _posts/2017/05/2017-05-04-windows-installer.md]: I think you mean add dates for these tasks? I can take a try, and if they aren't reasonable, feel free to change them.

**Comment by kariljordan** [comment]
@rgaiacs I tried making changes and submitting a PR on a separate branch, but kept getting errors. I committed directly to your branch. My apologies for that.

**Comment by rgaiacs** [comment]
> I tried making changes and submitting a PR on a separate branch, but kept getting errors. I committed directly to your branch. My apologies for that.

@kariljordan Thanks for the changes. Something happen when I try to merge you changes with my local changes and I end up with a not clean tree so I copy the post to a new branch and force a push to have a clean tree.

File: _posts/2017/05/2017-05-04-windows-installer.md (hunk 7 above)
Comment (rgaiacs) [reply _posts/2017/05/2017-05-04-windows-installer.md:9]: I added it at https://github.com/swcarpentry/website/pull/707/files#diff-f61ccb4a4bbbacc64b1655cf96a4d300R42.

File: _posts/2017/05/2017-05-04-windows-installer.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+   The installer will support Windows 7, Windows 8 and Windows 10.
+   The [end of extended support to Windows 7 by Microsoft is January 14, 2020](https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet)
+   and because we don't have data about how many learners are using Windows 7,
+   we believe it would be unfair to not include Windows 7
+   at the first release of our installer.
+
+4. Do we have a date where the new installer would be available?
+
+   Not yet. Software Carpentry staffs and Software Carpentry Steering Committee
```
Comment (rgaiacs) [inline _posts/2017/05/2017-05-04-windows-installer.md]: @jduckles @k8hertweck please check if the language is OK before merge this pull request.

**Comment by jduckles** [comment]
Everything looks good, but I'd like to drop the last point 5, it confuses more than it helps imo.

**Comment by rgaiacs** [comment]
I want to keep point 5 because some instructor are very motivated to try using other installer and I don't want to create confusion with instructions that are using HPC/Amazon/Azure/...
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Pull request merged**